jenkins/vsts-device-tests-set-status.sh

```diff
@@ -44,8 +44,6 @@ if test -z "$DEVICE_TYPE"; then
 	DEVICE_TYPE="iOS/tvOS"
 fi
 
-P=$(cat tmp.p)
```

Comment:
@rolfbjarne I don't know why that's breaking things :P 

Now it works: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3341819

Before it failed: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3338569

Reply:
Well ok so I guess it fails at `cat: tmp.p: No such file or directory` but I thought I saw other builds with that command failing and it still worked 🤷‍♂